server/models/notification.ts

```diff
@@ -0,0 +1,49 @@
+import dynamoose from 'dynamoose';
+
+export type NotificationType = {
+  id: string;
+  userID: string;
+  rideID: string;
```

Comment:
Sure!
However NotificationEvent is a type instead of a enum. Any way to convert this?

Reply:
oops thought it was an enum
since NotificationEvent is just the union of `Change` and `Status`, you can do something like `{ ...Object.values(Change), ...Object.values(Status) }`

Reply:
I think I've done something similar!

Reply:
ah sorry, didn't see that. looks good!

Reply:
Could you please approve again? Some formatting issue prevents merge.